doc/release-notes.md

```diff
@@ -66,6 +66,13 @@ New RPCs
 Build System
 ------------
 
+Files
+-----
+
+* On startup, the list of banned hosts and networks (via `setban` RPC) in
+  `banlist.dat` is ignored. Bitcoin Core version 22.x is the only version that
```

Comment:
(if I'm understanding your intention here)
```suggestion
  `banlist.dat` is ignored. Bitcoin Core version 22.x is the last version that
```

Reply:
it's the last version that can read .dat and the only version that can read .dat _and_ write to .json, so you're both right :-) Maybe just split it into two separate clauses?